apps/starter/src/ensemble/dashboard.yaml

```diff
@@ -265,14 +265,149 @@ View:
         - Text:
             id: myText
             text: Peter Parker
-        - Button:
-            label: Click Me
-            onTap:
-              executeCode: myText.setText("Spiderman")
         - Text:
             text: ${myText.text}
         - Text:
             text: ${getDummyJson.body.total}
+        - Stepper:
+            id: stepper1
+            styles:
+              connectorColor: "#31C48D"
+              connectorHeight: 4
+            steps:
+              - stepLabel: Match
+                contentWidget:
+                  Column:
+                    crossAxis: center
+                    mainAxis: spaceBetween
+                    children:
+                      - Text:
+                          text: Hello
+                      - Button:
+                          label: Doing Good
+              - stepLabel: SubmitDocs
+                contentWidget:
+                  Column:
+                    crossAxis: center
+                    mainAxis: spaceBetween
+                    children:
+                      - Text:
+                          text: Doing asd
+                      - Button:
+                          label: Doing Good
+              - stepLabel: Check
+                contentWidget:
+                  Column:
+                    crossAxis: center
+                    mainAxis: spaceBetween
+                    children:
+                      - Text:
+                          text: Doing asd
+                      - Button:
+                          label: Doing Good
+            item-template:
+              data:
+                - name: Match
+                  iconName: ScaleOutlined
+                - name: SubmitDocs
+                  iconName: FindInPageOutlined
+                - name: Review
+                  iconName: Check
+                - name: Enroll
+                  iconName: Search
+              name: step
+              template:
+                Conditional:
+                  conditions:
+                    - if: ${index < stepsLength - 1}
+                      Column:
+                        crossAxis: center
+                        children:
+                          - Icon:
+                              name: ${step.iconName}
+                              color: ${!active ? completed ? "#31C48D" :"grey" :"#1D1C21"}
```

Comment:
Should be `step.active`?

Reply:
the active, completed states are exposed from within component, so yeah can me made as step.active but is it necessary @evshi .

Reply:
Yes, it makes it consistent with the rest of the expressions, otherwise it's unclear where the active variable is coming from.